create webpage according to the requirements for day 3

TamaraDimov opened the pull request:
-complete website with functional navigation bar that contains 3 different sections (list of books, add a new book, contact information) 
-updated ReadMe file with the new features

TamaraDimov committed:
create webpage according to the requirements for day 3

TamaraDimov committed:
edit readme and fix eslint errors

TamaraDimov committed:
update readme

GraceHtet committed:
Update CSS, JS and HTML

Updated JS
Updated HTML
Changed font and styling

GraceHtet committed:
Fix linter errors

Owusu-Desmond commented on js/app.js:
```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
+
+// nav bar
+/* eslint-disable no-unused-vars */
+const displaySection = function displaySection(section) {
+  const sectionP = document.getElementById('description');
+  const heading = document.getElementById('title');
+  const sectionList = document.getElementById('list');
```
- [ ] Please according to javascript best practices you should use the arrow function instead of using the `function` keyword :smiley:

Owusu-Desmond commented on js/app.js:
```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
+// Date and time
+const clockElement = document.getElementById('clock');
+
+function clock() {
+  clockElement.textContent = new Date().toString();
+}
```
- [ ] Please according to javascript best practices you should use the arrow function instead of using the `function` keyword :smiley:

Owusu-Desmond commented on js/app.js:
```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
+setInterval(clock, 1000);
+
+// nav bar
+/* eslint-disable no-unused-vars */
```
- [ ] Please you should not disable any eslint in your code. please make sure that you remove this line and fix what eslint is saying :blush:

Owusu-Desmond commented on js/app.js:
```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
+      break;
+  }
+};
+/* eslint-disable no-unused-vars */
```
- [ ] Please you should not disable any eslint in your code. please make sure that you remove this line and fix what eslint is saying :blush:

Owusu-Desmond commented on js/app.js:
```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
+  const sectionForm = document.getElementById('form');
+  const sectionContact = document.getElementById('contact');
+
+  switch (section) {
+    case 'list':
+      heading.innerHTML = 'All Awesome Books';
+      sectionList.style.display = 'block';
+      sectionForm.style.display = 'none';
+      sectionContact.style.display = 'none';
+      sectionP.style.display = 'none';
+      break;
+
+    case 'form':
+      heading.innerHTML = 'Add a New Book';
+      sectionList.style.display = 'none';
+      sectionForm.style.display = 'block';
+      sectionContact.style.display = 'none';
+      sectionP.style.display = 'none';
+      break;
+
+    case 'contact':
+      heading.innerHTML = 'Contact Information';
+      sectionList.style.display = 'none';
+      sectionForm.style.display = 'none';
+      sectionContact.style.display = 'block';
```
- [ OPTIONAL ] Why don't you use the add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods  instead of using this whole lot of code.

Owusu-Desmond reviewed (changes requested):
Hi Team 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/xUOxfh6ZM75efM3Bqo/giphy-downsized-medium.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GraceHtet committed:
Fixed required changes

Changed normal functions to arrow function
Changed the  condition with switch case.
Edited the click function for nav menus.

AmaduKamara reviewed (approved):
# Status: Approved 🟢 🟢 🟢

Hi @TamaraDimov,

Good job on fixing the changes highlighted by the previous reviewer, your app looks great 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ App navigation works fine as expected 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
